### ISSUE TITLE
Unable to prepare setup for platform pod_point.*: Platform not found (No module named 'podpointclient.charge_mode')

### ISSUE THREAD

**Issue by a-d-r-i-a-n-d** [mattrayner/pod-point-home-assistant-component#42]
## Version of the custom_component

1.2.0

## Configuration

```yaml

Add your logs here.

```

## Describe the bug
Unable to install the custom component through HACS `1.32.1` on a `Generic x86-64` hardware

## Debug log

```text

Logger: homeassistant.setup
Source: setup.py:379 
First occurred: 13:22:28 (4 occurrences) 
Last logged: 13:22:28

Unable to prepare setup for platform pod_point.binary_sensor: Platform not found (No module named 'podpointclient.charge_mode').
Unable to prepare setup for platform pod_point.sensor: Platform not found (No module named 'podpointclient.charge_mode').
Unable to prepare setup for platform pod_point.switch: Platform not found (No module named 'podpointclient.charge_mode').
Unable to prepare setup for platform pod_point.update: Platform not found (No module named 'podpointclient.charge_mode').

```

**Comment by mattrayner** [comment]
Hi @a-d-r-i-a-n-d I seem to have missed when your issue was created. Are you still seeing the issue? I would recommend re-installing the plugin, it seems that your home assistant instance has not installed 'podpointclient', a dependency which should automatically get installed. OI hope this helps!

**Comment by a-d-r-i-a-n-d** [comment]
Hi @mattrayner ,

Thanks for getting back, I do indeed still having this issue. I did try to re-install the plugin but it didn't worked. To give you some more context, I've been running Home Assistant in a kubernetes cluster (installed through a helm chart) for the last few years and had pod point integration working with no issues at all. I've then decided to migrate HA to its own separate hardware (an Asus ChromeBox 3 i7 855u) and the issue above appeared. I tried to enable debug logging but that doesn't give me more information than the one in my initial post. Please let me know if there is anything you'd like me to do to debug further and thanks for your help in advance. Happy new year!

**Comment by mattrayner** [comment]
Hi @a-d-r-i-a-n-d, can you see anywhere in the logs an attempt to install podpointclient using pip? If not, it might be worth installing it manually in your system? I run the Hassos image on a dedicated box and haven't been able to recreate this issue

**Comment by a-d-r-i-a-n-d** [comment]
I can't see an attempt to install podpointclient in the logs. I am not entirely sure how to install it manually. Here is a new related error after deleting the podpoint integration:

```
Logger: homeassistant.config_entries
Source: config_entries.py:557
First occurred: 20:13:31 (4 occurrences)
Last logged: 20:13:31

Error unloading entry a*****u@gmail.com for binary_sensor
Error unloading entry a*****u@gmail.com for sensor
Error unloading entry a*****u@gmail.com for switch
Error unloading entry a*****u@gmail.com for update
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 557, in async_unload
    result = await component.async_unload_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py", line 176, in async_unload_entry
    return await component.async_unload_entry(entry)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 203, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!
```

Could something persist somewhere (like maybe .storage) and the uninstall to be partially done?

**Comment by mattrayner** [comment]
Hi @a-d-r-i-a-n-d, I've released a new version of the component (2.0.1)

Can you try updating and see if the issue persists for you?

**Comment by a-d-r-i-a-n-d** [comment]
Hi @mattrayner,

Yuhuu, the issue is now gone after the last update. Thanks a lot, much appreciated!
